### ISSUE TITLE
investigate reported issues with quantized BatchNorm2d

### ISSUE THREAD

**Issue by vkuzo** [pytorch/pytorch#43774]
## 🐛 Bug

In https://discuss.pytorch.org/t/incorrect-results-after-loading-saved-quantized-model/93844, there is a report of incorrect behavior of unfused quantized BatchNorm2d.  We should verify that it works correctly.

## To Reproduce

1.
1.
1.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @anjali411 @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a @vkuzo

**Comment by vkuzo** [comment]
we should definitely still fix this.  There were more dependencies than expected on first attempt, will send another PR but unfortunately it will have to be 1.8.

**Comment by malfet** [comment]
Removing 1.9 milestone. 
@vkuzo please do not hesitate to add it back if it needs to be  in 1.9 release.

**Comment by vkuzo** [comment]
this is fixed by https://github.com/pytorch/pytorch/pull/70432
